### PR TITLE
Virtualize all info panel forms

### PR DESCRIPTION
**Pull request by akalin-keybase**
One step closer to unifying all of them.

### CODE REVIEW

File: shared/chat/conversation/info-panel/index.js [hunk 1]
```diff
@@ -147,11 +89,57 @@ type ParticipantRow = {
...
+type RowType = $PropertyType<TeamRow, 'type'>
+
+const _renderTeamRow = (i: number, props: TeamRow) => {
   switch (props.type) {
-    case 'header':
+    case 'conversation footer':
+      return (
+        <Box key={props.key} style={{...globalStyles.flexBoxColumn, alignItems: 'stretch'}}>
+          <Divider style={{marginBottom: 20, marginTop: 10}} />
+
```
Comment (chrisnojima) [inline shared/chat/conversation/info-panel/index.js]: lets get rid of these extra spaces, we don't have this elsewhere

**Comment by akalin-keybase** [comment]
It'll be one list eventually (in an upcoming PR)

File: shared/chat/conversation/info-panel/index.js (hunk 1 above)
Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/index.js]: Done (had to set the small property on the button bar, too)

**Pull request merged**